common/utils/src/main/resources/error/error-classes.json

```diff
@@ -125,6 +125,12 @@
     ],
     "sqlState" : "428FR"
   },
+  "CANNOT_ASSIGN_EVENT_TIME_COLUMN_WITHOUT_WATERMARK" : {
+    "message" : [
+      "Watermark needs to be defined to reassign event time column. Could not find watermark definition in the streaming query."
```

Comment:
Nit: `Failed to find watermark definition`

Reply:
Fixed.